### DISCUSSION TITLE
Set Base-Path to access Flowise UI

### DISCUSSION THREAD

**Discussion by PythonbergIT** [FlowiseAI/Flowise#2889] [Q&A]
We deployed Flowise in a Azure Container Instance. In order to access the Flowise UI, it would be necessarry to set the base path in the environment variables to "/flowise".

Motivation:
Currently the assets are not reachable, when trying to access the Flowise UI.
![image](https://github.com/user-attachments/assets/a32c5ed8-29f3-4159-ae7c-8a6a5a555646)

**Comment by jmm-usu** [comment]
+1

**Comment by MB-MuratBayraktar** [comment]
Looking forward for a solution to this problem

**Reply by tiarebalbi** [reply]
#5254
